src/core/generators/angular.ts

```diff
@@ -55,12 +56,12 @@ ${
   };
   observe?: any;
```

Comment:
you also removed the `context?: HttpContext;` by mistake I think?

Reply:
hard day... :S